packages/core/src/domain/configuration.ts

```diff
@@ -43,7 +43,7 @@ export interface UserConfiguration {
   internalMonitoringApiKey?: string
   allowedTracingOrigins?: Array<string | RegExp>
   sampleRate?: number
-  resourceSampleRate?: number
+  resourceSampleRate?: number // deprecated
```

Comment:
Why are we only deprecating it instead of removing it directly?

Reply:
it can currently be used by some customers, we should not break the behavior if they are relying on it

Reply:
Makes sense. Does that mean that we will always keep it? or we may remove it sometime in the future?

Reply:
As long as we provide the SDK with a CDN file that updates itself, it can be tricky to remove deprecated features.
However, some ongoing reflexion on extra monitoring could help us to clean up deprecated features at some point.